### ISSUE TITLE
Crash when saving profile

### ISSUE THREAD

**Issue by dangu** [texstudio-org/texstudio#2144]
Edit: Now using the template!

### Environment
- TeXstudio: TeXstudio 4.2.2 (git 4.2.2)
- Qt: Using Qt Version 5.15.2, compiled with Qt 5.15.2 R
- OS: Windows 10
- TeX distribution: MikTeX

### Expected behavior
Saving profile using Options/Save profile should work

### Actual behavior

When using Options/Save profile I got this:

![image](https://user-images.githubusercontent.com/783084/156548913-76bcb67b-ff7c-41f6-9b91-5083aae3f5ff.png)

Please see attached backtrace.

[texstudio_backtrace1.txt](https://github.com/texstudio-org/texstudio/files/8177131/texstudio_backtrace1.txt)

Note: The problem seems to occur only once. When trying to reproduce it now it works as expected.

### How to reproduce
- Upgrade from older TeXstudio (I think I was runninng 4.2.1 before?)
- Save profile using Options/Save profile

**Comment by sunderme** [comment]
I assume that you are using txs for win with qt5 and you used to have txs with qt6.
Let txs run and stop once and then it should work.
Any particular reason why to ignore the questions in the template ? Not like they are imperative for solving your problem ?

**Comment by dangu** [comment]
Oh, sorry. I see now that there is a template for the bug report. Somehow it didn't show up when I searched the web for how to report this issue.

**Comment by sunderme** [comment]
alright, has my advice worked ?
"I assume that you are using txs for win with qt5 and you used to have txs with qt6.
Let txs run and stop once and then it should work."

**Comment by dangu** [comment]
I guess it works, yes. It crashed once and now I cannot reproduce the problem. I don't know which qt versions txs has been using over the years. I just upgrade it every now and then so if older versions were using qt6 I guess that's the issue then.

**Comment by sunderme** [comment]
okay. Since I can't really fix the issue and txs/qt5 for win will hopefully stop soon, I leave it at that.

**Comment by octaeder** [comment]
as long as txs starts use from menu

![grafik](https://github.com/texstudio-org/texstudio/assets/102688820/82879d1c-db2b-4007-92a7-20e319d112c4)

The browser will open. Then press `New issue` button on the web page.
